src/api/handlers/action.py

```diff
@@ -150,7 +150,7 @@ def community_admin_list(self, request):
   @super_admins_only
   def super_admin_list(self, request): 
     context: Context = request.context
-    args: dict = context.args
+    #args: dict = context.args
```

Comment:
nit: if not needed remove

Reply:
This comment is about style, not substance.  Can save time by ignoring stuff that isn't important.